microcosm_elasticsearch/index_status/store.py

```diff
@@ -0,0 +1,56 @@
+"""
+Index Status Store
+
+"""
+from microcosm_elasticsearch.index_status.models import IndexStatus
+
+
+class IndexStatusStore:
+
+    def __init__(self, graph):
+        self.index_registry = graph.elasticsearch_index_registry
+
+    def process_status_data(self, status, stats, index):
+        """
+        There is a known structure to how ES returns data about itself
```

Comment:
Could not we just dump underlying json from elasticsearch rather than imposing structure with resources?  These are not business objects that we need to guard with type.

Reply:
hmm we probably could, I guess I assumed that this would make it more human readable but we would then need to update in case ES changed their field structure. Ok, ill expand it to just return everything.

Reply:
That's what I was thinking; we'd have to track upstream changes for purely informational data..